R/modules.R

```diff
@@ -280,7 +281,7 @@ root_modules <- function(...) {
 #' runApp(app)
 #' }
 module <- function(label = "module",
-                   server = function(id, data) {
+                   server = function(id, data, filter_panel_api) {
```

Comment:
In the stop starting on line 305 you should also refer to filter_panel_api ?

Reply:
![Screenshot from 2022-07-12 11-17-02](https://user-images.githubusercontent.com/86970066/178456116-90338805-43e8-479d-aa72-e5cc2934381d.png)